sklearn/preprocessing/_encoders.py

```diff
@@ -449,7 +449,8 @@ def fit(self, X, y=None):
 
         Returns
         -------
-        self
+        self:
```

Comment:
When there is no type associated, the `:` is not necessary https://numpydoc.readthedocs.io/en/latest/format.html#parameters 
```suggestion
        self
```